### PR TITLE
add test-bed and xdp description

### PR DESCRIPTION
**Pull request by williamtu**

### CODE REVIEW

File: doc/lpc18/background.tex [hunk 1]
```diff
@@ -176,7 +176,21 @@ \subsubsection{Concurrency}
...
 \subsection{XDP: eXpress data path}
 
-TODO.
+The core idea behind XDP is optimizing cases where relatively simple
+decisions can be made about incoming packets. It allows the loading
+of a BPF program into the kernel; that program gets an opportunity
+to inspect packets before they enter the networking stack itself.
+The initial use case for XDP was to enable the quick dropping of
+unwanted packets, but it has since expanded to cover simple routing
+decisions and packet modifications; see this in-progress documentation
```
Comment (mbudiu-vmw) [inline doc/lpc18/background.tex]: are you missing a citation here?

File: doc/lpc18/background.tex [hunk 2]
```diff
@@ -176,7 +176,21 @@ \subsubsection{Concurrency}
 
 \subsection{XDP: eXpress data path}
 
-TODO.
+The core idea behind XDP is optimizing cases where relatively simple
```
Comment (mbudiu-vmw) [inline doc/lpc18/background.tex]: I think that you are repeating this phrase in the next paragraph to a large degree. Perhaps you should only keep that one.

File: doc/lpc18/background.tex (hunk 1 above)
Comment (williamtu) [reply doc/lpc18/background.tex]: yes, I will fix it

File: doc/lpc18/background.tex (hunk 2 above)
Comment (williamtu) [reply doc/lpc18/background.tex]: ok! thanks

**Pull request closed without merging**